### ISSUE TITLE
RequiredFields includes part of locale code in error message

### ISSUE THREAD

**Issue by robbieaverill** [tractorcow/silverstripe-fluent#216]
Given that I have a Fluent enabled Page like this:

```php
class MyObject extends Page
{
    private static $db = ['MyFieldHere' => 'Varchar'];

    public function getCMSValidator()
    {
        return new RequiredFields(['MyFieldHere']);
    }
}
```

... when I submit the CMS form, the error message says `"enMy Field Here" is required`:

![Required field included locale prefix](https://cloud.githubusercontent.com/assets/5170590/16635941/300de2ee-4429-11e6-8df6-790dbc49017a.png)

**SS:** 3.4.0
**Fluent:** 3.5.0

**Comment by tractorcow** [comment]
Oh, that's quite odd. It must be because of the label tag appearing in the message.

I can't see an easy way around this, other than maybe adjusting the tag so that it encodes nicer? Maybe `"(en) Block Name" is required` would be acceptable?

**Comment by robbieaverill** [comment]
Yeah, it didn't look simple when I had a quick look at it. That suggestion would definitely be an improvement!

**Comment by tractorcow** [comment]
The problem is the [en] flag would then have round brackets on it too. This is why it's hard to put custom html into field labels. :)
